pandas/tests/test_import.py

```diff
@@ -0,0 +1,14 @@
+def test_import_error():
+    # GH#26367 Verify whether Pandas import succeeds when setting filterwarnings
+    import warnings
+
+    warnings.filterwarnings("error")
```

Comment:
Let's also test `'always'` as mentioned in the issue.

Reply:
Done